AudioManagerUtilities/src/CAmSocketHandler.cpp

```diff
@@ -118,17 +138,20 @@ void CAmSocketHandler::start_listenting()
     {
         if (mRecreatePollfds)
         {
+            #ifdef WITH_TIMERFD
```

Comment:
Don't indent #ifdef.
(or in case you think that style is OK, it's still not consistent with previous #ifdefs)